### PR TITLE
Implement DRK spell Absorb-Attri

### PR DESCRIPTION
**Pull request by CatsEyeXI**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?
Implements Black Magic Spell: Absorb-Attri
Will be used in implementation of Trust: Zeid II

## Steps to test these changes
image

### CODE REVIEW

**Comment by TeoTwawki** [comment]
Would you mind updating the comment [here](https://github.com/LandSandBoat/server/blob/24a4e3f2f1832686353d467a42d7d1f7783a7c2c/scripts/enum/msg.lua#L98) from saying `1` to saying `<number>` as part of this PR?

**Comment by TeoTwawki** [comment]
also the scroll data to learn this spell seems to be missing

File: scripts/enum/msg.lua [hunk 1]
```diff
@@ -95,7 +95,7 @@ xi.msg.basic =
     MAGIC_ABSORB_MND       = 334, -- <caster> casts <spell>. <target>'s MND is drained.
     MAGIC_ABSORB_CHR       = 335, -- <caster> casts <spell>. <target>'s CHR is drained.
     MAGIC_ERASE            = 341, -- <caster> casts <spell>. <target>'s <status> effect disappears!
-    MAGIC_STEAL            = 430, -- <caster> casts <spell>. 1 of <target>'s effects is drained.
+    MAGIC_STEAL            = 430, -- <caster> casts <spell>. <number> of <target>'s effects is drained.
```
Comment (zach2good) [inline scripts/enum/msg.lua:98]: This also needs to be added to `message_basic.h`

**Review by zach2good — COMMENTED** [review]
Almost ready 👍

File: scripts/enum/msg.lua (hunk 1 above)
Comment (CatsEyeXI) [reply scripts/enum/msg.lua:98]: added

**Review by zach2good — APPROVED** [review]

**Pull request merged**